index.html

```diff
@@ -142,53 +183,58 @@ <h2 class="about-sub-head">Let's Connect</h2>
                         <li><a href="#"><img src="icon/git.svg"></a></li>
                     </ul>
                 </div>
-                     <button class="resume-btn">Get My resume</button>
+                   <button class="resume-btn">Get My resume</button>
             </div>
+
+            
             <div class="language-header">
-                <h3 class="language-heading">Languages<i class="fa fa-angle-down"></i></i></h3>
-              </div>
-              <ul class="list-language">
-                  <li class="for-web"><img src="icon/js.png" alt="">JavaSript</li>
-                  <li class="for-web"><img src="icon/html.png" alt="">HTML</li>
-                  <li class="for-web"><img src="icon/css.png" alt="">CSS</li>
-              </ul> 
-              <ul class="frame-work">
-                  <li class="skill">
-                      <h3 class="first-title">Frameworks<i class="fa fa-angle-right" ></i></h3>
-                      <hr class="line">  
-                  </li>
-                  <li class="skills">
-                      <h3 class="second-title">Skills<i  class="fa fa-angle-right" ></i></h3> 
-                      <hr class="line">
-                  </li>
-              </ul>
-             </section>
 
-             <section class="form">
+                <h3 class="language-heading">Languages <img src="images/down.png" alt="" srcset=""></i></h3>
+
+            <ul id="for-desk" class="list-language">
+                <li class="for-web"><img src="icon/js.png" alt=""><p>JavaSript</p></li>
+                <li class="for-web"><img src="icon/html.png" alt=""><p>HTML</p></li>
+                <li class="for-web"><img src="icon/css.png" alt=""><p>CSS</p></li>
```

Comment:
- [ ] Great job implementing the missing icons requested by the previous reviewer, but I think there are still some design requirements you missed. 😄 
- - The icons are not properly aligned as it's been shown in the Figma design, Please you need to look into this as your project is meant to look like the Figma design.
- Your current design:
![Screenshot 2023-04-01 201733](https://user-images.githubusercontent.com/107632780/229311861-ba1b5565-cef2-43c7-8b1e-0341d7304db6.png)

- Figma design:
![Screenshot 2023-04-01 205638](https://user-images.githubusercontent.com/107632780/229311886-35bbb23d-96c3-4639-a165-4ffd18ac72db.png)